### PR TITLE
Switch check targets from npm to bun

### PR DESCRIPTION
**Pull request by jberdine**
bun is much faster (about 17 vs 26 sec for make check-all in my setup)

### CODE REVIEW

**Comment by jberdine** [comment]
This will fail CI until we next update the container images, so draft for now.

**Comment by jberdine** [comment]
This is good to go now. Anyone have comments/concerns?

File: skipruntime-ts/Makefile [hunk 1]
```diff
@@ -6,27 +6,27 @@ SCRIPT_DIR=$(shell dirname $(shell realpath $(firstword $(MAKEFILE_LIST))))
...
-check-all: build check-src check-examples
+check-all: bunbuild check-src check-examples
 
 .PHONY: build
 build:
 	cd .. && npm install && npm run build
 
 .PHONY: bunbuild
 bunbuild:
-	cd .. && bum install && bun run build
```
Comment (beauby) [inline skipruntime-ts/Makefile:29]: Oopsie

**Review by beauby — APPROVED** [review]

File: skipruntime-ts/Makefile (hunk 1 above)
Comment (mbouaziz) [reply skipruntime-ts/Makefile:29]: Bummer!

**Pull request merged**